homeworks/yehor.honcharenko_NOcode-developer/GamesProject/About.html

```diff
@@ -0,0 +1,96 @@
+                    <img class="logoTop" src="./Images/logo-ma.svg" width="64px" height="64px" alt="logotype master of code academy">
+                    <ul class="nav__ul">
+                        <li class="nav__li">
+                            <a class="nav_link" href="Cards.html">Games</a>
+                        </li>
+                        <li class="nav__li">
+                            <a class="nav_link" href="#">About</a>
+                        </li>
+                    </ul>
+                </nav>
+            </header>
+            <main class="content">
+                <section class="content__container">
+                    <article class="content__img">
+                        <img src="./Images/image-about1.svg" alt="image about us" style="width: 100%;">
+                    </article>
+                    <article class="content__about">
+                        <article class="about__header">
+                            <h1 class="header__title">About me</h1>
+                        </article>
+                        <article class="about__text">
+                            <p class="text__paragraph-black">
+                                Hi! My name is Yehor and I'm a Junior Frontend
+                                Developer. I am already familiar with main Web
+                                Technologies like HTML, CSS, JavaScript and Git version
+                                control system.
+                            </p>
+                            <p class="text__paragraph-black">
+                                This page was developed during the course
+                                <a class="link__paragraph-black" href="https://www.mastersacademy.education/frontend-for-beginners-it"
+                                    target="_blank">'Frontend for beginners' from Masters Academy in
+                                    2023</a>.
+                            </p>
+                            <p class="text__paragraph-black">
+                                This is a social project from MOCG company where I got
+                                an opportunity to work with Frontend mentors and to
+                                create my own small project for the portfolio.
+                            </p>
+                            <p class="text__paragraph-black">
+                                You can contact me via <a class="link__paragraph-black" href="https://www.linkedin.com/in/yehor-honcharenko/"
+                                    target="_blank">linkedin</a> and/or
+                                check out my <a class="link__paragraph-black" href="https://github.com/NOcode-developer"
+                                    target="_blank">GitHub</a>.
+                            </p>
+                        </article>
+                    </article>
+                </section>
+            </main>
+            <footer>
+                <section class="footer__container">
```

Comment:
By BEM methodology you always should start from the block. The main idea is to catch BEM-block with all elements and paste it to another page. It should work as a separate structure.
So, proly, this is the block `footer`

Reply:
I attempted to fix this, but I'm uncertain if it's the correct solution